web/test/pages/roadmap.test.tsx

```diff
@@ -68,15 +68,21 @@ describe("roadmap page", () => {
   const renderRoadmapPage = ({
     sidebarDisplayContent,
     operateReferences,
+    fundings,
+    certifications,
   }: {
     sidebarDisplayContent?: Record<string, SidebarCardContent>;
     operateReferences?: Record<string, OperateReference>;
+    fundings?: Funding[];
+    certifications?: Certification[];
```

Comment:
these don't need to be part of the arguments if they're never being used by a test. we can just pass `[]` always on line 84/85. same for the function below as well.